tests/test_cli.py

```diff
@@ -0,0 +1,85 @@
+import click
+from click.testing import CliRunner
+from getgist.__main__ import run_getgist, run_getmy
+from getgist.__main__ import run_putgist, run_putmy, run_lsgists, run_mygists
+from getgist.__main__ import GETGIST_DESC, GETMY_DESC, PUTGIST_DESC, PUTMY_DESC, LSGISTS_DESC, MYGISTS_DESC
```

Comment:
Why three lines importing from the same module?

```suggestion
from getgist.__main__ import (
    GETGIST_DESC,
    GETMY_DESC,
    LSGISTS_DESC,
    MYGISTS_DESC,
    PUTGIST_DESC,
    PUTMY_DESC,
    run_getgist,
    run_getmy,
    run_lsgists,
    run_mygists,
    run_putgist,
    run_putmy,
)
```